The flag 'directory' used in rule aggregate_transcript_analysis is only valid for outputs, not inputs.
The flag 'directory' used in rule output_combine_files is only valid for outputs, not inputs.
Building DAG of jobs...
Using shell: /bin/bash
Provided cores: 32
Rules claiming more threads will be scaled down.
Conda environments: ignored
Job counts:
	count	jobs
	1	all
	1	gene_filter
	1	output_combine_files
	3
Select jobs to execute...

[Mon Apr  5 11:54:23 2021]
checkpoint gene_filter:
    input: /project/owlmayerTemporary/Sid/ForOlga/Sequence/dhx9.fa
    output: Results/Genes_temp
    jobid: 2
Downstream jobs will be updated after completion.

[Mon Apr  5 11:54:25 2021]
Error in rule gene_filter:
    jobid: 2
    output: Results/Genes_temp

RuleException:
CalledProcessError in line 460 of /home/annaldas/projects/IsoTV/Snakefile:
Command 'set -euo pipefail;  /home/annaldas/miniconda3/envs/isotv_update/bin/python3.7 /project/owlmayerTemporary/Sid/ForOlga/Sequence/.snakemake/scripts/tmpz7f0nt5s.filter_genes.py' returned non-zero exit status 1.
  File "/home/annaldas/miniconda3/envs/isotv_update/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2347, in run_wrapper
  File "/home/annaldas/projects/IsoTV/Snakefile", line 460, in __rule_gene_filter
  File "/home/annaldas/miniconda3/envs/isotv_update/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 568, in _callback
  File "/home/annaldas/miniconda3/envs/isotv_update/lib/python3.7/concurrent/futures/thread.py", line 57, in run
  File "/home/annaldas/miniconda3/envs/isotv_update/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 554, in cached_or_run
  File "/home/annaldas/miniconda3/envs/isotv_update/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2359, in run_wrapper
Shutting down, this might take some time.
Exiting because a job execution failed. Look above for error message
Complete log: /home/annaldas/projects/IsoTV/.snakemake/log/2021-04-05T115410.889020.snakemake.log
